validator/src/main.rs

```diff
@@ -30,12 +30,9 @@ use {
         validator::{is_snapshot_config_valid, Validator, ValidatorConfig, ValidatorStartProgress},
     },
     solana_gossip::{cluster_info::Node, contact_info::ContactInfo},
-    solana_ledger::{
-        blockstore_db::DEFAULT_ROCKS_FIFO_SHRED_STORAGE_SIZE_BYTES,
```

Comment:
It looks like you moved the constant from `blockstore_db` to `blockstore_options`; this seems fine, but this PR should then also delete the constant from `blockstore_db` right?

Reply:
Nice catch.  I thought I've removed it but turns out I didn't.  Fixed.